.github/workflows/build.yml

```diff
@@ -187,3 +189,30 @@ jobs:
         run: make init
       - name: Run functional & smoke tests
         run: pytest -vv -n 4 tests/functional tests/smoke
+
+  docker-disabled:
+    name: docker-disabled-test
+    if: github.repository_owner == 'aws'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: |
+            3.7
+            3.8
+            3.9
+            3.10
+      - name: Init samdev
+        run: make init
+      - name: Stop Docker
+        run: sudo systemctl stop docker
+      - name: Check Docker not Running
+        run: docker info
+        id: run-docker-info
+        continue-on-error: true
+      - name: Report failure
+        if: steps.run-docker-info.outcome == 'success'
+        run: exit 1
+      - name: Run tests without Docker
+        run: pytest -vv tests/integration/buildcmd/test_build_cmd.py -k TestBuildCommand_PythonFunctions_WithoutDocker
```

Comment:
1. Should this be a make command at all?
2. Should we be running this with `SAM_CLI_DEV=1` like https://github.com/aws/aws-sam-cli/blob/develop/Makefile#L20?

Reply:
1. Not too sure what you mean, it's not currently a make command.
2. 👍

Reply:
On one: I was asking if this should be a make command. Not a big deal.

Reply:
Gotcha. Don't think it needs to be. We have other tests in our actions we run directly through pytest. Don't think we need to abstract them all with make.